src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/DeleteOrganisationIntTest.java

```diff
@@ -19,12 +19,12 @@
 import static uk.gov.hmcts.reform.professionalapi.helper.OrganisationFixtures.someMinimalOrganisationRequest;
 
 @RunWith(SpringIntegrationSerenityRunner.class)
```

Comment:
We need to remove all the @RunWith and use @ExtendWith instead

Reply:
Addressed here: https://github.com/hmcts/rd-professional-api/pull/1122/commits/711ccd95c9dc836ae3880e50cc47aa46ddf8ab1b